composer.lock

```diff
@@ -1796,24 +1818,24 @@
                     "type": "tidelift"
                 }
             ],
-            "time": "2021-08-16T04:20:12+00:00"
+            "time": "2021-11-28T23:30:39+00:00"
         },
         {
             "name": "psr/cache",
-            "version": "2.0.0",
+            "version": "1.0.1",
```

Comment:
Are you sure about it?

Reply:
I was using php 7.4 to update composer. Should it be php8?